src/client/Microsoft.Identity.Client/AppConfig/AbstractApplicationBuilder.cs

```diff
@@ -295,6 +295,24 @@ public T WithExtraQueryParameters(string extraQueryParameters)
             }
             return (T)this;
         }
+
+        /// <summary>
+        /// Allows usage of experimental features and APIs. If this flag is not set, experimental features 
+        /// will throw an exception. 
+        /// </summary>
+        /// <remarks>
+        /// Changes in the public API of experimental features will not result in an increment of the major version of this library.
+        /// The same level of support is given to experimental features as to all other features.
+        /// Affected APIs are clearly documented as being experimental. 
+        /// </remarks>
```

Comment:
></remarks> [](start = 12, length = 10)

Use of experimental features in production is not recommended and is subject to be removed between builds

Reply:
Unless you are directed to, use of experimental features ... ?